### PR TITLE
Add to_ieee754_64 Presto Scalar Function

### PR DESCRIPTION
**Pull request by pdabre12**
Fixes to_ieee754_64 Presto function to return 64 bits instead of 64 bytes.

Fixes: https://github.com/prestodb/presto/issues/20704

### CODE REVIEW

File: velox/functions/prestosql/tests/BinaryFunctionsTest.cpp [hunk 1]
```diff
@@ -590,56 +590,47 @@ TEST_F(BinaryFunctionsTest, toIEEE754Bits64) {
   };
 
   EXPECT_EQ(std::nullopt, toIEEE754Bits64(std::nullopt));
-  EXPECT_EQ(
-      "0100000000100100100110011001100110011001100110011001100110011010",
+  EXPECT_EQ(hexToDec("0000000000000000"), toIEEE754Bits64(0.0));
+  EXPECT_EQ(hexToDec("3FF0000000000000"), toIEEE754Bits64(1.0));
+  EXPECT_EQ(hexToDec("8000000000000001"), toIEEE754Bits64(-5e-324));
+  EXPECT_EQ(hexToDec("0000000000000000"), toIEEE754Bits64(0.0));
+  EXPECT_EQ(hexToDec("3FF0000000000000"), toIEEE754Bits64(1.0));
```
Comment (aditi-pandit) [inline velox/functions/prestosql/tests/BinaryFunctionsTest.cpp:597]: The tests for values 0.0 and 1.0 are repeated.

**Comment by pdabre12** [comment]
@aditi-pandit  Fixed the issues.

File: velox/functions/prestosql/BinaryFunctions.h [hunk 2]
```diff
@@ -415,9 +415,10 @@ struct ToIEEE754Bits64 {
   FOLLY_ALWAYS_INLINE void call(
       out_type<Varbinary>& result,
       const arg_type<double>& input) {
-    result.resize(64);
-    bits::toString((const uint64_t*)&input, 0, 64, result.data());
-    std::reverse(result.data(), result.data() + 64);
+    static constexpr auto kTypeLength = sizeof(int64_t);
```
Comment (aditi-pandit) [inline velox/functions/prestosql/BinaryFunctions.h:418]: This code is exactly the same as ToBigEndian64 as above. I would have thought that you have to memcpy the double into a bit array and then do a reverse like the previous code.

**Review by aditi-pandit — APPROVED** [review]
Thanks !

**Comment by aditi-pandit** [comment]
@Yuhta : Please can you review.

**Review by aditi-pandit — CHANGES_REQUESTED** [review]
Please run the fuzzer and paste its output https://facebookincubator.github.io/velox/develop/testing/fuzzer.html#how-to-run

**Comment by pdabre12** [comment]
@aditi-pandit 

**Fuzzer output for to_ieee754_64 Presto Scalar function:**

I20230907 15:46:38.054050 1173282 ExpressionVerifier.cpp:63] ===================
I20230907 15:46:38.054075 1173282 ExpressionVerifier.cpp:74] All results match.
I20230907 15:46:38.054101 1173282 ExpressionFuzzer.cpp:1358] ==============================> Done with iteration 7776
I20230907 15:46:38.054133 1173282 ExpressionFuzzer.cpp:1093] ==============================> Top 1 by number of rows processed
I20230907 15:46:38.054148 1173282 ExpressionFuzzer.cpp:1095] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230907 15:46:38.054162 1173282 ExpressionFuzzer.cpp:1099] to_ieee754_64 7777 100.00% 125961
I20230907 15:46:38.054184 1173282 ExpressionFuzzer.cpp:1105] ==============================> Bottom 1 by number of rows processed
I20230907 15:46:38.054199 1173282 ExpressionFuzzer.cpp:1107] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230907 15:46:38.054214 1173282 ExpressionFuzzer.cpp:1112] to_ieee754_64 7777 100.00% 125961
I20230907 15:46:38.054231 1173282 ExpressionFuzzer.cpp:1125] ==============================> All stats sorted by number of times the function was chosen
I20230907 15:46:38.054246 1173282 ExpressionFuzzer.cpp:1127] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20230907 15:46:38.054261 1173282 ExpressionFuzzer.cpp:1131] to_ieee754_64 7777 100.00% 125961
[==========] Running 0 tests from 0 test suites.
[==========] 0 tests from 0 test suites ran. (0 ms total)
[  PASSED  ] 0 tests.

**Review by aditi-pandit — APPROVED** [review]

**Pull request closed without merging**